auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -347,9 +346,7 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
                 });
 
         // Facebook sign out
-        if (FacebookSdk.isInitialized()) {
```

Comment:
@samtstern Since you noticed, I might as well explain it now instead of in my review. The easiest would be to read Facebook's release notes: https://developers.facebook.com/docs/android/change-log-4x#4_19_0 and the upgrade guide https://developers.facebook.com/docs/android/upgrading-4x#4180to4190.